with-formik/package.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Nice job getting the pkg json right 👍

Reply:
The credit goes to you for that. The documentation is clear and expo-cli helps where in doubt :)